tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -692,3 +692,68 @@ def forward_xp(self, inputs, xp):
 def test_swap_invalid(xp, shape, axis1, axis2):
     a = array_utils.create_dummy_ndarray(xp, shape, 'float32')
     return xp.swapaxes(a, axis1, axis2)
+
+
+@op_utils.op_test(['native:0'])
+@chainer.testing.parameterize_pytest('shape', [
+    (2, 2, 2),
+    (3, 3, 2, 3, 3),
+    (3, 0, 2, 0, 3),
+    (1, 2, 3, 1, 3, 3),
+    (3, 4, 5, 2, 3, 5),
+    (1,)
+])
+class TestExpandDIms(op_utils.NumpyOpTest):
+
+    # TODO(kshitij12345): Remove this when fixed
+    check_numpy_strides_compliance = False
+
+    def setup(self, dtype):
+        # Skip backward/double-backward tests for int dtypes
```

Comment:
PR #6621 seems merged soon. Could you remove this skip after #6621 is merged?

Reply:
Sure, will address that once it is in.

Reply:
It will take a little more time to merge #6621, so I'll merge this PR firstly.

Reply:
Sure, thank you.